geotrek/tourism/templatetags/tourism_tags.py

```diff
@@ -33,3 +35,11 @@ def touristic_content_categories():
 @register.simple_tag
 def is_tourism_enabled():
     return settings.TOURISM_ENABLED
+
+
+@register.simple_tag
+def places_coords():
+    places = TouristicEventPlace.objects.annotate(
+        geom_4326=Transform('geom', 4326)
+    ).all()
```

Comment:
```suggestion
    )
```
all is redundant here